Kontent.Ai.Management/IManagementClient.cs

```diff
@@ -803,4 +804,23 @@ public interface IManagementClient
     /// <param name="previewConfiguration">Represents configuration that will be used for project.</param>
     /// <returns>The <see cref="PreviewConfigurationModel"/> instance that represents the preview configuration.</returns>
     Task<PreviewConfigurationModel> ModifyPreviewConfigurationAsync(PreviewConfigurationModel previewConfiguration);
+
+    /// <summary>
+    /// Activates the web spotlight status with possibility to provide existing Root Type ID.
```

Comment:
I would adjust the wording here a bit:

`Activates the web spotlight, allowing you to specify an existing Root Type ID.`

what do you think?

Reply:
Done.